roles/openshift_control_plane/tasks/main.yml

```diff
@@ -256,9 +256,11 @@
   retries: 60
   delay: 5
   with_items:
-  - "{{ 'etcd' if inventory_hostname in groups['oo_etcd_to_config'] else omit }}"
+  - "{{ 'etcd' if (inventory_hostname in groups['oo_etcd_to_config'] and inventory_hostname in groups['oo_masters_to_config']) else '' }}"
```

Comment:
and inventory_hostname in groups['oo_masters_to_config'] is redundant;  This play only executes against hosts in that group.

Reply:
You're right, the bz clearly shows that it was attempting to look for a pod named `__omit_place_holder__ee64c3e15fab51456d4a1fd4ea054f1dc27b73d6`
Updated